src/mono/wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="3.7.0" />
```

Comment:
For future PR, we should consider using the common version (more than one!) property for this, being used in other places in the repo.